tensorflow/java/src/main/native/operation_builder_jni.cc

```diff
@@ -257,3 +257,32 @@ JNIEXPORT void JNICALL Java_org_tensorflow_OperationBuilder_setAttrShape(
   TF_SetAttrShape(d, cname, cvalue.get(), static_cast<int>(num_dims));
   env->ReleaseStringUTFChars(name, cname);
 }
+
+JNIEXPORT void JNICALL Java_org_tensorflow_OperationBuilder_setAttrStringList(
+        JNIEnv* env, jclass object, jlong handle, jstring name, jobjectArray values) {
+ static_assert(sizeof(jbyte) == 1,
+                "Require Java byte to be represented as a single byte");
+  TF_OperationDescription* d = requireHandle(env, handle);
+  if (d == nullptr) return;
+  const char *cname = env->GetStringUTFChars(name, nullptr);
```

Comment:
Style nit: To be consistent with the styling used in the rest of this file, `const char* cname` instead of `const char *cname` etc.

Would you mind running [`clang-format`](http://clang.llvm.org/docs/ClangFormat.html) on C/C++ source code and [`google-java-format`](https://github.com/google/google-java-format) on Java source code to ensure consistent formatting?